src/bynder-js-sdk.js

```diff
@@ -38,15 +38,19 @@ class APICall {
      * Create a APICall.
      * @constructor
      * @param {string} baseURL - A string with the base URL for account.
+     * @param {string} httpsAgent - A https agent.
+     * @param {string} httpAgent - A http agent.
      * @param {string} url - A string with the name of the API method.
      * @param {string} method - A string with the method of the API call.
      * @param {Object} consumerToken - An object with both the public and secret consumer keys.
      * @param {Object} accessToken - An object with both the public and secret access keys.
      * @param {Object} [data={}] - An object containing the query parameters.
      */
-    constructor(baseURL, url, method, consumerToken, accessToken, data = {}) {
+    constructor(baseURL, httpsAgent, httpAgent, url, method, consumerToken, accessToken, data = {}) {
```

Comment:
These changes, the way they are implemented, will bring incompatibilities with previous versions of the SDK.
Please move the new parameters to the end of the function declaration.

Reply:
Well, actually nope, somehow I ignored the rest of the file where you add the agent to *all* the calls 😄
You could indeed have added it to the end of the declaration but in theory it won't cause any problem.